ansible/configs/controller-migration-casc/README.adoc

```diff
@@ -0,0 +1,50 @@
+= ansible-automation-platform config
+
+Author: Eric Lavarde <elavarde@redhat.com>
+Owner: Eric Lavarde <elavarde@redhat.com>
+Alternative Contacts: Goetz Rieger <grieger@redhat.com>
+
+== Overview
+
+This config is meant to be generic and reusable by others, but has been primarily developed
+to cover the needs of the Summit labs (formerly known as):
+- Getting Started with Ansible Tower
+- Ansible Tower Advanced Lab.
+
+== Setup
+
+Copy the `sample_vars.yml` to a place outside of the Git repo (for security reasons)
+and adapt the variables to your needs. You can also add and overwrite variables
+present in the link:env_vars.yml[env_vars.yml file].
+
+TIP: you'll need to adapt _once_ all variables marked with `CHANGEME` to adapt them to your own person.
+The variables marked with `CHANGEME-ec2` need to be adapted once for each new AWS-sandbox deployment (as long as you keep the same sandbox, they don't need to change).
+The GUID only needs to be different if you deploy the same environment multiple times in the same sandbox.
+
+== Running Ansible Playbook
+
+From the `agnosticd/ansible` directory run:
```

Comment:
can you specify which Execution Environment should be used here? Is it ee-multicloud?